dandi/dandiarchive.py

```diff
@@ -175,7 +175,7 @@ class _dandi_url_parser:
                 rf"(/files(\?location=(?P<location>.*)?)?)?"
             ),
             {"server_type": "api"},
-            "https://gui-beta-dandiarchive-org.netflif.app/#/dandiset"
+            "https://gui-beta-dandiarchive-org.netlify.app/#/dandiset"
```

Comment:
bugaga -- so this regex was of no value and it still worked?  may be we should just get rid of this record altogether?

Reply:
It's not a regex; it's part of the string displayed to humans when none of the regexes match.

Reply:
d'oh - right.  I will blame the narrow context of the diff for my slip. Thanks!